pages/api/submit.ts

```diff
@@ -108,6 +109,21 @@ const previewNotify = async (form: PublicFormSchemaProperties, fields: Responses
     });
 };
 
+const deleteFiles = (files: formidable.Files) => {
+  for (const [_key, value] of Object.entries(files)) {
```

Comment:
Eslint doesn't like this for some reason. Would just use a blank https://www.samanthaming.com/tidbits/13-skip-values-in-destructuring/